gui/velociraptor/src/components/artifacts/artifacts.jsx

```diff
@@ -275,6 +275,7 @@ class ArtifactInspector extends React.Component {
                           onClick={() => this.setState({showNewArtifactDialog: true})}
                           variant="default">
                     <FontAwesomeIcon icon="plus"/>
+		    <span className="sr-only">Add an Artifact</span>
```

Comment:
Thanks for this PR - it is very important! 

We need to add translation to the messages so we need to call T() as in line 272.

In fact the tooltips we use are set up in line 272 and I wonder if there is a way to have that automatically populate with the sr-only class? Then we won't need to redefine it again.

The tooltips are added in css here https://github.com/Velocidex/velociraptor/blob/19d84ad127ad4c3919db0c94b3604f793c6a864f/gui/velociraptor/src/css/index.css#L95

Reply:
Good call out, thank you.

I do have a question about the translations. One word I want to add "Search" isn't included in the translation files. I see "Search" is used in conjunction with other phrases, would it be safe to just use what is provided there? Also where should I add the search pairs in the language file? Does the organization matter, or is it okay to include it at the bottom of each language file?

As for automatically populating the label and tooltips, it's possible by extending the bootstrap component to account for this. It wouldn't be hard to map tooltip prop value to be part of the span element. My worry is potentially breaking some functionality else where. Is it alright to just use spans inside the button for now and maybe account for a new button component in a future PR?

Reply:
We try to keep all the language files in sync but they are probably not :-). 

The way the translation works is that if there is no translation in the relevant language file it just emits the actual string used. So in practice I don't worry about adding the translations because it will just be in English for missing ones.

Then I do a single pr to update all the languages later. There is a script that uses Google translate to get some basic translation up.